### PR TITLE
Main

### PR DESCRIPTION
**Pull request by Nyle-FBM**
Nyle's part for static webpages

### CODE REVIEW

**Comment by Nyle-FBM** [comment]
swag

File: package.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "react-icons": "^4.4.0"
```
Comment (rubendplaza) [inline package.json:3]: @Nyle-FBM Does this have to be here or can it be moved to /client/package.json?

Comment (Nyle-FBM) [reply package.json:3]: tbh idek. but all that's for is the pdf symbol which i'll probably change anyway

Comment (Nyle-FBM) [reply package.json:3]: yes

Comment (rubendplaza) [reply package.json:3]: okay no worries lol i was just tryna be nice. it actually should be in the client folders package json. what you likely did was an "npm install react-icons" inside "patient-management-system" folder but it needed to be done in the client folder. All you gotta do is delete the package json and package lock json files and do the "npm install react-icons" inside client

Comment (rubendplaza) [reply package.json:3]: * delete the json files from the "patient-management-system" folder

Comment (rubendplaza) [reply package.json:3]: i mean

**Pull request closed without merging**